lib/final-api/v1/http/ddtf_build.rb

```diff
@@ -16,6 +16,19 @@ def initialize(build, options = {})
           @request = build.request
         end
 
+        def map_build_state
```

Comment:
build_state2api_v1status and put it as a constant: `BUILD_STATE2API_V1STATUS` on the beginning